lib/internal/webstreams/adapters.js

```diff
@@ -440,10 +463,7 @@ function newReadableStreamFromStreamReadable(streamReadable, options = kEmptyObj
   streamReadable.pause();
 
   const cleanup = finished(streamReadable, (error) => {
-    if (error?.code === 'ERR_STREAM_PREMATURE_CLOSE') {
-      const err = new AbortError(undefined, { cause: error });
-      error = err;
-    }
+    error = handleKnownInternalErrors(error);
```

Comment:
Instead of this, why don't we mutate it inside the function?

```suggestion
    handleKnownInternalErrors(error);
```

Reply:
Because it's much more code to

- remove the message
- set the code
- set the cause
- change the name
- change the prototype

compared to instantiating what we need, the way we want it.

Reply:
Also, it's not something you can handle the null case with.